EPRN.Portal/ViewModels/PRNS/ActionPrnViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using EPRN.Common.Enums;
+using EPRN.Portal.Resources;
+using System.ComponentModel.DataAnnotations;
+namespace EPRN.Portal.ViewModels.PRNS
+{
+    public class ActionPrnViewModel
+    {
+        public int JourneyId { get; set; }
```

Comment:
This needs to be renamed Id, and it does not need setting as a hidden input in the view. It will be populated from the route during the binding. (I am just removing the others from all models and views at the moment)

Reply:
Done.